src/olympia/devhub/templates/devhub/new-landing/components/footer.html

```diff
@@ -7,8 +7,9 @@ <h3 class="DevHub-Footer-sections-header">{{ _('Resources') }}</h3>
         <h4>{{ _('Documentation') }}</h4>
         <ul>
           <li><a href="https://developer.mozilla.org/Add-ons">{{ _('Add-ons') }}</a></li>
-          <li><a href="https://developer.mozilla.org/Add-ons/WebExtensions">{{ _('WebExtensions') }}</a></li>
+          <li><a href="https://developer.mozilla.org/Add-ons/WebExtensions">{{ _('Extensions') }}</a></li>
           <li><a href="https://developer.mozilla.org/Add-ons/Themes/Background">{{ _('Themes') }}</a></li>
```

Comment:
@atsay Do we want to use this opportunity to add the language slug to MDN urls? If not, users get redirected to their locale on MDN and that page might or might not exist or be up to date.